modules/homestakeros/default.nix

```diff
@@ -223,7 +223,7 @@ in
               Type = "simple";
             };
 
-            script = ''${pkgs.go-ethereum}/bin/go-ethereum \
+            script = ''${pkgs.go-ethereum}/bin/geth \
```

Comment:
there is also this helper: `lib.getExe`. see, e.g: https://github.com/jhvst/nix-config/blob/main/home-manager/juuso.nix#L149